amqplib/index.d.ts

```diff
@@ -1,13 +1,77 @@
-// Type definitions for amqplib 0.3.x
+// Type definitions for amqplib 0.5.x
 // Project: https://github.com/squaremo/amqp.node
-// Definitions by: Michael Nahkies <https://github.com/mnahkies>, Ab Reitsma <https://github.com/abreits>
+// Definitions by: Michael Nahkies <https://github.com/mnahkies>, Ab Reitsma <https://github.com/abreits>, Nicolás Fantone <https://github.com/nfantone>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-/// <reference types="when" />
 /// <reference types="node" />
 
-declare module "amqplib/properties" {
-    namespace Replies {
+declare module 'amqplib' {
+    import * as Promise from 'bluebird';
+    import * as events from 'events';
+    import shared = require('amqplib/properties');
+    import callback_api = require('amqplib/callback_api');
+
+    export import Replies = shared.Replies;
```

Comment:
My mistake, I thought that exporting in the fashion I suggested also imported them into the local scope